### PR TITLE
Bta 1134: Vat Var activation

### PR DESCRIPTION
**Pull request by AndrewShanks**

### CODE REVIEW

File: app/services/EnrolmentsStoreService.scala [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import models.{UserEnrolmentStatus, UserEnrolments, VatEnrolment, VatVarEnrolment}
+import org.joda.time.{DateTime, DateTimeZone}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class EnrolmentStoreServiceImpl @Inject()(connector: EnrolmentStoreConnector)(implicit val ec:ExecutionContext) extends EnrolmentsStoreService {
+
+  val daysBetweenExpectedArrivalAndExpiry = 23
```
Comment (gianick) [inline app/services/EnrolmentsStoreService.scala]: "Magic number". Shall we keep this in the conf file?

File: app/views/partials/account_summary/vat/vat_var/prompt_to_activate_no_new_pin.scala.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+@import config.FrontendAppConfig
+
+@(emacUrlBuilder: EmacUrlBuilder, vatDecEnrolment: VatDecEnrolment, appConfig: FrontendAppConfig, currentUrl: String,
+  forCard: Boolean)(implicit request: Request[_], messages:Messages)
+
+@if(forCard){
+  <h3 id="change-vat-details-header">@messages("vat.vat_var_for_card.change_vat_details_online")</h3>
+} else {
+  <h2 id="change-vat-details-header">@messages("vat.vat_var_for_card.change_vat_details_online")</h2>
+}
```
Comment (gianick) [inline app/views/partials/account_summary/vat/vat_var/prompt_to_activate_no_new_pin.scala.html]: Should they not be the same? `<h2>` or `<h3>`?

File: conf/application.conf [hunk 3]
```diff
@@ -98,6 +98,11 @@ microservice {
         port = 9732
       }
 
+      enrolment-store{
```
Comment (gianick) [inline conf/application.conf]:
```suggestion
      enrolment-store {
```

File: app/views/partials/account_summary/vat/vat_var/prompt_to_activate_no_new_pin.scala.html (hunk 2 above)
Comment (AndrewShanks) [reply app/views/partials/account_summary/vat/vat_var/prompt_to_activate_no_new_pin.scala.html]: An h3 is consistent with the rest of the card design, an h2 with the rest of the subpage design.

File: app/services/EnrolmentsStoreService.scala (hunk 1 above)
Comment (AndrewShanks) [reply app/services/EnrolmentsStoreService.scala]: I don't know, it seems very cumbersome to put a value like that in the conf file when it's only used in one place and makes little sense out of context - I already had to give it a horrible name in context.

**Comment by AndrewShanks** [comment]
Relevant changes merged on alternative branch

**Pull request closed without merging**